library/oss/postgres/prepare/database/cleanup_functions.sql

```diff
@@ -0,0 +1,7 @@
+-- should clean up all functions that are created by functions.sql
```

Comment:
At least add a comment to functions.sql (to remember updating this).

An alternative would be to define a go struct with turnup or turndown instructions.

Reply:
You mean, move this out of a data file and define it in go?

could even be non-ugly with raw string literals

Reply:
yep

Reply:
done